kptncook/mealie.py

```diff
@@ -32,70 +41,71 @@ class RecipeIngredient(BaseModel):
     note: str | None
     unit: UnitFoodBase | None
     food: UnitFoodBase | None
-    disable_amount: bool = True
-    quantity: float = 1
+    disableAmount: bool = True
+    quantity: float | None
 
 
 class RecipeSummary(BaseModel):
     id: UUID4 | None
 
-    user_id: UUID4 | None = Field(None, alias="userId")
-    group_id: UUID4 | None = Field(None, alias="groupId")
+    userId: UUID4 | None = Field(None, alias="userId")
```

Comment:
```python
class RecipeSummary(BaseModel):
    user_id: str | None = Field(None, alias="userId")
    group_id: str | None = Field(None, alias="groupId")

> > >rs = RecipeSummary(userId="foo", groupId="bar")
> > >print(rs.user_id, rs.group_id)
foo bar

> > >print(rs.dict(by_alias=True))
{'userId': 'foo', 'groupId': 'bar'}
```
Hmm, I don't really get why camel case is used here..

Reply:
iirc the reason why I changed it, was, because the API has thrown errors for snake case variables in json. As I searched some time to nail it down (the mealie api-documenation is not the best), I just changed every json related varibale (I admit - instead of using alias as changing that was faster).